### DISCUSSION TITLE
Area limitations

### DISCUSSION THREAD

**Discussion by JJStobart** [UMEP-dev/UMEP#530] [Q&A]
Hello Solweig users! I was hoping to get a general idea on anyone that has used Solweig on a broad scale (i.e. greater than 100km2 area)? Has anyone tried running the model over large areas and if so did you come across any issues or any ideas of model run times etc. 
Big thanks in advance, 
Joe

**Comment by biglimp** [comment]
We often use SOLWEIG on an urban scale. To accomplish this you need to tile your data and run smaller areas. Otherwise you will end up with memory issues. We use either the Model Designer in the Processing Toolbox or automate the process using a Python script.

**Reply by JJStobart** [reply]
thank you so much for the quick reply! I was thinking that even on powerful machines one is still effectivly processing large areas of multiple rasters - i will go with a tiled approach
